compiler/src/dotty/tools/dotc/reporting/ErrorMessageID.scala

```diff
@@ -1,187 +1,190 @@
-    MissingEmptyArgumentListID,
-    DuplicateNamedTypeParameterID,
-    UndefinedNamedTypeParameterID,
-    IllegalStartOfStatementID,
-    TraitIsExpectedID,
-    TraitRedefinedFinalMethodFromAnyRefID,
-    PackageNameAlreadyDefinedID,
-    UnapplyInvalidNumberOfArgumentsID,
-    UnapplyInvalidReturnTypeID,
-    StaticFieldsOnlyAllowedInObjectsID,
-    CyclicInheritanceID,
-    BadSymbolicReferenceID,
-    UnableToExtendSealedClassID,
-    SymbolHasUnparsableVersionNumberID,
-    SymbolChangedSemanticsInVersionID,
-    UnableToEmitSwitchID,
-    MissingCompanionForStaticID,
-    PolymorphicMethodMissingTypeInParentID,
-    ParamsNoInlineID,
-    JavaSymbolIsNotAValueID,
-    DoubleDefinitionID,
-    MatchCaseOnlyNullWarningID,
-    ImportRenamedTwiceID,
-    TypeTestAlwaysDivergesID,
-    TermMemberNeedsNeedsResultTypeForImplicitSearchID,
-    ClassCannotExtendEnumID,
-    ValueClassParameterMayNotBeCallByNameID,
-    NotAnExtractorID,
-    MemberWithSameNameAsStaticID,
-    PureExpressionInStatementPositionID,
-    TraitCompanionWithMutableStaticID,
-    LazyStaticFieldID,
-    StaticOverridingNonStaticMembersID,
-    OverloadInRefinementID,
-    NoMatchingOverloadID,
-    StableIdentPatternID,
-    StaticFieldsShouldPrecedeNonStaticID,
-    IllegalSuperAccessorID,
-    TraitParameterUsedAsParentPrefixID,
-    UnknownNamedEnclosingClassOrObjectID,
-    IllegalCyclicTypeReferenceID,
-    MissingTypeParameterInTypeAppID,
-    SkolemInInferredID,
-    ErasedTypesCanOnlyBeFunctionTypesID,
-    CaseClassMissingNonImplicitParamListID,
-    EnumerationsShouldNotBeEmptyID,
-    IllegalParameterInitID,
-    RedundantModifierID,
-    TypedCaseDoesNotExplicitlyExtendTypedEnumID,
-    IllegalRedefinitionOfStandardKindID,
-    NoExtensionMethodAllowedID,
-    ExtensionMethodCannotHaveTypeParamsID,
-    ExtensionCanOnlyHaveDefsID,
-    UnexpectedPatternForSummonFromID,
-    AnonymousInstanceCannotBeEmptyID,
-    TypeSpliceInValPatternID,
-    ModifierNotAllowedForDefinitionID,
-    CannotExtendJavaEnumID,
-    InvalidReferenceInImplicitNotFoundAnnotationID,
-    TraitMayNotDefineNativeMethodID,
-    JavaEnumParentArgsID,
-    AlreadyDefinedID,
-    CaseClassInInlinedCodeID,
-    OverrideTypeMismatchErrorID,
-    OverrideErrorID,
-    MatchableWarningID,
-    CannotExtendFunctionID,
-    LossyWideningConstantConversionID,
-    ImplicitSearchTooLargeID
+  case EmptyCatchOrFinallyBlockID extends ErrorMessageID(isActive = false) // errorNumber: 0
+  case EmptyCatchBlockID // errorNumber: 1
+  case EmptyCatchAndFinallyBlockID // errorNumber: 2
+  case DeprecatedWithOperatorID // errorNumber: 3
+  case CaseClassMissingParamListID // errorNumber: 4
+  case DuplicateBindID // errorNumber: 5
+  case MissingIdentID // errorNumber: 6
+  case TypeMismatchID // errorNumber: 7
+  case NotAMemberID // errorNumber: 8
+  case EarlyDefinitionsNotSupportedID // errorNumber: 9
+  case TopLevelImplicitClassID extends ErrorMessageID(isActive = false) // errorNumber: 10 
```

Comment:
As far as I know this is no longer an error, however there is a message that uses this and it's actually referenced in `Desugar.scala`

https://github.com/lampepfl/dotty/blob/53f5531e7e31eca02b3db158c8c384e713b4542a/compiler/src/dotty/tools/dotc/ast/Desugar.scala#L755

But I'm unable to ever get it to fall into that branch.